src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -2661,5 +2661,53 @@ class C
         return x;
     }
 }");
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsExtractMethod)]
+        public Task TestFlowNullable_LambdaWithReturn()
+            => TestInRegularAndScriptAsync(
```

Comment:
in a future PR, could you update everything to be consistent here :)

Reply:
Naming style, or the expression body style?

Reply:
expression-body style.